### PR TITLE
Update minio version in docker compose files

### PR DESCRIPTION
**Pull request by dbampalikis**
The minio versions on the docker compose files are very (around 2 years) old and it don't run locally (at least for me). The PR updates the versions of the minio instance and uses the new way of passing credentials.

The version used here has been tested in sda-dashboard

### CODE REVIEW

**Review by nanjiangshu — APPROVED** [review]
Looks good to me!

**Review by blankdots — APPROVED** [review]

File: dev_utils/compose-no-tls.yml [hunk 1]
```diff
@@ -41,19 +41,25 @@ services:
...
-    command: server /data
+    command: server /data --console-address ":9001"
     container_name: s3
     environment:
-      - MINIO_ACCESS_KEY=access
-      - MINIO_SECRET_KEY=secretkey
+      - MINIO_NOTIFY_ELASTICSEARCH_ENABLE=on
+      - MINIO_NOTIFY_ELASTICSEARCH_URL=http://elasticsearch:9200
+      - MINIO_NOTIFY_ELASTICSEARCH_INDEX=minio
+      - MINIO_NOTIFY_ELASTICSEARCH_FORMAT=namespace
```
Comment (aaperis) [inline dev_utils/compose-no-tls.yml:50]: Why are these needed?

File: dev_utils/compose-sda.yml [hunk 2]
```diff
@@ -72,19 +72,25 @@ services:
...
-    command: server /data
+    command: server /data --console-address ":9001"
     container_name: s3
     environment:
-      - MINIO_ACCESS_KEY=access
-      - MINIO_SECRET_KEY=secretkey
+      - MINIO_NOTIFY_ELASTICSEARCH_ENABLE=on
+      - MINIO_NOTIFY_ELASTICSEARCH_URL=http://elasticsearch:9200
+      - MINIO_NOTIFY_ELASTICSEARCH_INDEX=minio
+      - MINIO_NOTIFY_ELASTICSEARCH_FORMAT=namespace
```
Comment (aaperis) [inline dev_utils/compose-sda.yml:81]: Same as above.

File: dev_utils/compose-no-tls.yml (hunk 1 above)
Comment (dbampalikis) [reply dev_utils/compose-no-tls.yml:50]: They are not actually, local testing leftovers. Removing them

**Review by jbygdell — APPROVED** [review]
